test/pageobjects/Camera.js

```diff
@@ -1,11 +1,15 @@
 import BasePage from './BasePage.js'
 import { verifyElementCopy } from '../utils/mochaw'
 import { assert } from 'chai'
+import { browserName } from '../main'
 
 class Camera extends BasePage {
   async enableCameraButton() {
     return this.$('[data-onfido-qa="enable-camera-btn"]')
   }
+  async allowCameraPermissionImage() {
+    return this.$('div.onfido-sdk-ui-CameraPermissions-Primer-graphic')
```

Comment:
Should be able to just specify the class name here.
```suggestion
    return this.$('.onfido-sdk-ui-CameraPermissions-Primer-graphic')
```